subprojects/gdk_rust/gdk_electrum/src/servers-mainnet.txt

```diff
@@ -0,0 +1,62 @@
+81-7-10-251.blue.kundencontroller.de:50002:s:noverify
+E-X.not.fyi:50002:s:noverify
+VPS.hsmiths.com:50002:s:noverify
+b.ooze.cc:50002:s:noverify
+bitcoin.corgi.party:50002:s:noverify
+bitcoins.sk:50002:s:noverify
+btc.cihar.com:50002:s:noverify
+btc.xskyx.net:50002:s:noverify
+currentlane.lovebitco.in:50002:s:noverify
+electrum.jochen-hoenicke.de:50005:s:noverify
+dragon085.startdedicated.de:50002:s:noverify
+e-1.claudioboxx.com:50002:s:noverify
+e.keff.org:50002:s:noverify
+electrum-server.ninja:50002:s:noverify
+electrum-unlimited.criptolayer.net:50002:s:noverify
+electrum.eff.ro:50002:s:noverify
+electrum.festivaldelhumor.org:50002:s:noverify
+electrum.hsmiths.com:50002:s:noverify
+electrum.leblancnet.us:50002:s:noverify
+electrum.mindspot.org:50002:s:noverify
+electrum.qtornado.com:50002:s:noverify
+electrum.taborsky.cz:50002:s:noverify
+electrum.villocq.com:50002:s:noverify
+electrum2.eff.ro:50002:s:noverify
+electrum2.villocq.com:50002:s:noverify
+electrumx.bot.nu:50002:s:noverify
+electrumx.ddns.net:50002:s:noverify
+electrumx.ftp.sh:50002:s:noverify
+electrumx.soon.it:50002:s:noverify
+elx01.knas.systems:50002:s:noverify
+fedaykin.goip.de:50002:s:noverify
+fn.48.org:50002:s:noverify
+electrum.emzy.de:50002:s:noverify
+ndnd.selfhost.eu:50002:s:noverify
+orannis.com:50002:s:noverify
+rbx.curalle.ovh:50002:s:noverify
+tardis.bauerj.eu:50002:s:noverify
+technetium.network:50002:s:noverify
+tomscryptos.com:50002:s:noverify
+ulrichard.ch:50002:s:noverify
+vmd27610.contaboserver.net:50002:s:noverify
+vmd30612.contaboserver.net:50002:s:noverify
+xray587.startdedicated.de:50002:s:noverify
+yuio.top:50002:s:noverify
+bitcoin.dragon.zone:50004:s:noverify
+ecdsa.net:110:s:noverify
+btc.usebsv.com:50006:s:noverify
+e2.keff.org:50002:s:noverify
+electrum.hodlister.co:50002:s:noverify
+electrum3.hodlister.co:50002:s:noverify
+electrum5.hodlister.co:50002:s:noverify
+electrumx.electricnewyear.net:50002:s:noverify
+fortress.qtornado.com:443:s:noverify
+green-gold.westeurope.cloudapp.azure.com:56002:s:noverify
+electrumx.erbium.eu:50002:s:noverify
+electrumx-core.1209k.com:50002:s:noverify
+bitcoin.aranguren.org:50002:s:noverify
+bitcoin.lukechilds.co:50002:s:noverify
+electrum.aantonop.com:50002:s:noverify
+electrum.bitkoins.nl:50512:s:noverify
+electrum.blockstream.info:50002:s:noverify
+blockstream.info:700:s:noverify
```

Comment:
These are not ordered, so I guess they were obtained with a old version of the script, but I cannot verify them.

Also they change from different Electrum versions, we might want to set such version somewhere for verifiers

Reply:
Yes, I added sorting at a later stage without updating the server list.

I updated it now to the Electrum v4.0.9 server list. I mentioned the Electrum version in the commit message, do you think its sufficient?

Reply:
Yes, sure